src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -3837,6 +3837,63 @@ public LazyTryStatement(Lazy<IBlockStatement> body, Lazy<ImmutableArray<ICatchCl
     }
 
     /// <summary>
```

Comment:
This is currently a hand crafted file, but @heejaechang is working on a generator (similar to our bound node and syntax node generator based on an xml). @jinujoseph - I think we need to check that in soon to avoid extra manual work in coding and reviews and also prevent unintentional coding errors in the generated file. Do we plan to enable this soon?